blocks/block-alignment-toolbar/index.js

```diff
@@ -3,11 +3,7 @@
  */
 import { __ } from '@wordpress/i18n';
 import { Toolbar } from '@wordpress/components';
-
-/**
- * Internal dependencies
- */
-import { withEditorSettings } from '../editor-settings';
+import { withSelect } from '@wordpress/data';
```

Comment:
Several components in the blocks module make use of these settings. These components should be moved to the `editor` module (related #6275) before merging this PR to ensure `select( 'core/editor' )` is available.